### PR TITLE
fix: Send message with "enter" also do new line

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

Fixes https://github.com/chatwoot/chatwoot/issues/5622

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## How Has This Been Tested?

1. Enable enter to send messages in profile settings.
2. And write "something" to test in rich editor.
3. And move the cursor to "someth | ing" and press enter key.
4. See the sent message.
5. Now it will be "someth ing" before it will break the line "someth
                                                                                                   ing"

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/components/widgets/WootWriter/Editor.vue [hunk 1]
```diff
@@ -278,6 +287,11 @@ export default {
         clearTimeout(this.idleTimer);
       }
     },
+    handleEnterKeyWhenEnterToSendEnabled(event) {
```
Comment (nithindavid) [inline app/javascript/dashboard/components/widgets/WootWriter/Editor.vue:290]: Found a bug on the help center article editor - https://www.loom.com/share/11f005fa91144b8cb9e089d7c38e4b37

**Review by nithindavid — CHANGES_REQUESTED** [review]

File: app/javascript/dashboard/components/widgets/WootWriter/Editor.vue [hunk 2]
```diff
@@ -58,13 +61,14 @@ const createState = (content, placeholder, plugins = []) => {
...
   components: { TagAgents, CannedResponse },
-  mixins: [eventListenerMixins],
+  mixins: [eventListenerMixins, uiSettingsMixin],
   props: {
     value: { type: String, default: '' },
     editorId: { type: String, default: '' },
     placeholder: { type: String, default: '' },
     isPrivate: { type: Boolean, default: false },
     enableSuggestions: { type: Boolean, default: true },
+    isHelpCenter: { type: Boolean, default: false },
```
Comment (nithindavid) [inline app/javascript/dashboard/components/widgets/WootWriter/Editor.vue:71]:
```suggestion
    overrideLineBreaks: { type: Boolean, default: false },
```

**Review by nithindavid — CHANGES_REQUESTED** [review]

File: app/javascript/dashboard/components/widgets/WootWriter/Editor.vue (hunk 2 above)
Comment (iamsivin) [reply app/javascript/dashboard/components/widgets/WootWriter/Editor.vue]: Done https://github.com/chatwoot/chatwoot/pull/5961/commits/f7d98e6ce4c9196dd41827f6f38c6f2419230596

**Review by fayazara — APPROVED** [review]
Working as expected tested locally.

https://user-images.githubusercontent.com/15716057/205561530-4a895f16-94c5-4c4f-80c6-5821b18d62dd.mp4

**Pull request merged**